diagnostic.md

```diff
@@ -35,5 +41,7 @@ Record your responses inside the fenced code blocks below each question.
     pass it data?
 
     ```html
-    <!-- your response here -->
+    {{ #each contact.phone as |phone| }}
+      {{ my-contact/my-phone phone=phone }}
+    {{/each}}
```

Comment:
You are close.  We need to include the sub-component as `my-phone`, also `contact.phones` will be plural.
```
{{#each contact.phones as |phone|}} 
{{my-phone phone=phone}}
{{/each}}
```